controls/cis_rhel10.yml

```diff
@@ -0,0 +1,3113 @@
+---
+policy: 'DRAFT - CIS Benchmark for Red Hat Enterprise Linux 10'
+title: 'DRAFT - CIS Benchmark for Red Hat Enterprise Linux 10'
+id: cis_rhel10
+version: 'Draft'
+source: https://www.cisecurity.org/cis-benchmarks/#red_hat_linux
+levels:
+  - id: l1_server
+  - id: l2_server
+    inherits_from:
+      - l1_server
+  - id: l1_workstation
+  - id: l2_workstation
+    inherits_from:
+      - l1_workstation
+reference_type: cis
+product: rhel10
+
```

Comment:
Have you checked if there are any rules that aren't applicable to RHEL 10 ?

Reply:
I used RHEL 9 as reference and actually discovered some rules incorrectly removed in RHEL 9 profiles. It will be addressed in another PR. Regarding RHEL 10, so far I didn't find anything that caught my attention but in any case this is an experimental profile. We will be more sure when the product and respective policy are released.